cypress/e2e/firstTest.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('First Test', () => {
+
+    it('Check the page', () => {
+        cy.visit('http://localhost:4200/pages/layout/stepper');
+        cy.contains("Step content #1");
```

Comment:
Pls, ensure that the contains is not verification. You have to use .should() or expect